storage/Devices/LuksImpl.cc

```diff
@@ -291,9 +291,25 @@ namespace storage
 
     void
     Luks::Impl::do_rename_in_etc_crypttab(const Actiongraph::Impl& actiongraph,
-				       const Device* lhs) const
+					  const Device* lhs) const
     {
-	// TODO
+	const Storage& storage = actiongraph.get_storage();
+
+	const Luks* luks_lhs = to_luks(lhs);
+
+	// TODO, error handling and mount-by
+
+	EtcFstab fstab(storage.get_impl().prepend_rootprefix("/etc"));	// TODO pass as parameter
+
+	FstabKey key(luks_lhs->get_blk_device()->get_name(), "");
+	FstabChange entry;
+	entry.device = get_blk_device()->get_name();
+	entry.dentry = get_name();
+	entry.encr = EncryptionType::LUKS;
+
+	fstab.removeEntry(key);
```

Comment:
Just an observation. Maybe, it would be more clear to use the same object type for both, `EtcFstab#removeEntry` and `EtcFstab#addEntry`. That is, use key or entry in both methods:
```
fstab.removeEntry(entry);
fstab.addEntry(entry);
```
or
```
fstab.removeEntry(key);
fstab.addEntry(key, entry);
```
Another option could be to add a method `EtcFstab#removeEntryByKey`.